Test Results - Average Loss: 0.5151, Average Dice Score: 0.7595100%[Finetuning] Epoch: 1 Avg Loss: 0.3134[Validation] Avg Loss: 0.3326, Avg Dice Score: 0.8946 100% ||[Finetuning] Epoch: 2 Avg Loss: 0.3017[Validation] Avg Loss: 0.3355, Avg Dice Score: 0.8850100%[Finetuning] Epoch: 3 Avg Loss: 0.2999[Validation] Avg Loss: 0.3307, Avg Dice Score: 0.8990100%[Finetuning] Epoch: 4 Avg Loss: 0.2981[Validation] Avg Loss: 0.3367, Avg Dice Score: 0.8985100%[Finetuning] Epoch: 5 Avg Loss: 0.2948[Validation] Avg Loss: 0.3320, Avg Dice Score: 0.8948100%[Finetuning] Epoch: 6 Avg Loss: 0.2913[Validation] Avg Loss: 0.3328, Avg Dice Score: 0.8888100%[Finetuning] Epoch: 7 Avg Loss: 0.2883[Validation] Avg Loss: 0.3361, Avg Dice Score: 0.8833100%[Finetuning] Epoch: 8 Avg Loss: 0.2882[Validation] Avg Loss: 0.3318, Avg Dice Score: 0.8960100%[Finetuning] Epoch: 9 Avg Loss: 0.2844[Validation] Avg Loss: 0.3430, Avg Dice Score: 0.8730100%[Finetuning] Epoch: 10 Avg Loss: 0.2845 [Validation] Avg Loss: 0.3356, Avg Dice Score: 0.8871143/143 [00:13<00:00, 10.50it/s]143/143 [00:13<00:00, 10.56it/s]143/143 [00:13<00:00, 10.48it/s]143/143 [00:13<00:00, 10.61it/s]143/143 [00:13<00:00, 10.57it/s]143/143 [00:13<00:00, 10.42it/s]143/143 [00:13<00:00, 10.50it/s]143/143 [00:13<00:00, 10.64it/s]143/143 [00:13<00:00, 10.34it/s]143/143 [00:13<00:00, 10.35it/s]/home/mikhelson/MedImSeg-Lab24/CLUE/lib/python3.8/site-packages/lightning/pytorch/core/module.py:447: You are trying to 'self.log()' but the 'self.trainer reference is not registered on the model yet. This is most likely because the model hasn't been passed to the 'Trainer'Test Results - Average Loss: 0.5984, Average Dice Score: 0.7156